NEWS.md

```diff
@@ -24,7 +24,7 @@ New library functions
 Standard library changes
 ------------------------
 
-* Cmd interpolation (``` `$(x::Cmd) a b c` ``` where) now propagates `x`'s process flags (environment, flags, working directory, etc) if `x` is the first interpolant and errors otherwise ([#24353]).
```

Comment:
Bad rebase?

Reply:
Good catch. Fixed in https://github.com/JuliaLang/julia/pull/31874

Reply:
Ooops, sorry about that!